alignak/objects/timeperiod.py

```diff
@@ -804,49 +811,46 @@ def resolve_daterange(self, dateranges, entry):
         res = re.search(r'([a-z]*) ([\d-]+) ([a-z]*)[\s\t]*([0-9:, -]+)', entry)
         if res is not None:
             # print "Good catch 11"
-            (t0, swday_offset, t1, other) = res.groups()
+            (t0, t2, t1, other) = res.groups()
             if t0 in Daterange.weekdays and t1 in Daterange.months:
-                swday = t0
-                smon = t1
+                swday = Daterange.get_weekday_id(t0)
+                smon = Daterange.get_month_id(t1)
                 emon = smon
                 ewday = swday
-                ewday_offset = swday_offset
+                ewday_offset = t2
                 dateranges.append(
-                    MonthWeekDayDaterange(0, smon, 0, swday, swday_offset, 0, emon,
+                    MonthWeekDayDaterange(0, smon, 0, swday, t2, 0, emon,
                                           0, ewday, ewday_offset, 0, other)
                 )
                 return
-
-        res = re.search(r'([a-z]*) ([\d-]+)[\s\t]+([0-9:, -]+)', entry)
-        if res is not None:
-            # print "Good catch 12"
-            (t0, smday, other) = res.groups()
-            if t0 in Daterange.weekdays:
-                swday = t0
-                swday_offset = smday
-                ewday = swday
-                ewday_offset = swday_offset
-                dateranges.append(
-                    WeekDayDaterange(0, 0, 0, swday, swday_offset, 0,
-                                     0, 0, ewday, ewday_offset, 0, other)
-                )
-                return
-            if t0 in Daterange.months:
-                smon = t0
-                emon = smon
-                emday = smday
-                dateranges.append(
-                    MonthDateDaterange(
-                        0, smon, smday, 0, 0, 0, emon, emday, 0, 0, 0, other)
-                )
-                return
-            if t0 == 'day':
-                emday = smday
-                dateranges.append(
-                    MonthDayDaterange(0, 0, smday, 0, 0, 0,
-                                      0, emday, 0, 0, 0, other)
-                )
-                return
+            if not t1:
+                # print "Good catch 12"
+                if t0 in Daterange.weekdays:
+                    swday = Daterange.get_weekday_id(t0)
+                    swday_offset = t2
+                    ewday = swday
+                    ewday_offset = swday_offset
```

Comment:
Looks like swday_offset is not defined here. Should be t2 right? 
Strange this variale was used from the first regex :(